### ISSUE TITLE
Updates to Events page

### ISSUE THREAD

**Issue by hlthomps** [trinitywebmedia/Norwescap-Tickets#55]
Please delete all events from September and October from page

Please add the text in the attached Word document at the top of the page

Under "Save the Date," please delete the following text:
Virtual Red Bow Raffle
November 15-30, 2021
Our annual tradition continues with an online “tricky tray” raffle for dozens of beautiful prize baskets! All proceeds benefit Norwescap’s Food Bank. Stay tuned for more details!

Please fix spacing for NorwesFest listing under Save the Date to eliminate blank lines in between the name of the event, the date, and the location

Thank you!

[2021 Nov 15 Basket Raffle language for website.docx](https://github.com/trinitywebmedia/Norwescap-Tickets/files/7539418/2021.Nov.15.Basket.Raffle.language.for.website.docx)

**Comment by trinitywebmedia** [comment]
Thank you Heather. I will delete and add to the page as noted. Also, it's worth mentioning again, that this site has a calendar that automatically deletes old events and has a much better User Experience than a blog post. I opened up a ticket a while ago about this issue with Kasia, which was then closed because of no response.

Is this something we should re-visit to make the Event experience better? -- GT
